guides/common/modules/proc_generating-a-report-for-installable-and-applicable-errata.adoc

```diff
@@ -0,0 +1,15 @@
+[id="Generating_a_Report_for_Installable_and_Applicable_Errata_{context}"]
+= Generating a Report for Installable and Applicable Errata
+
+Use the following procedure to view installable and applicable errata on managed hosts.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Monitor* > *Report Templates*.
+. Click *Generate* for the *Host {endash} Applicable Errata* template.
+. In the *Installability* drop down menu, select *Applicable* to show all applicable errata or *Installable* to limit the report exclusively to errata that are accessible in the Content View environments of your host that can be installed.
+. Optional: To schedule a report, click the icon to the right of the *Generate at* field and choose the date and time you want for the generated report.
+. Optional: To send a report to an e-mail address, select the *Send report via e-mail* checkbox, and in the *Deliver to e-mail addresses* field, enter the required e-mail address.
```

Comment:
For these two Optional steps, I would suggest not to describe the UI in so much detail.

. Optional: To schedule a report, use the *Generate at* field.
. Optional: To send a report to an e-mail address, select the *Send report via e-mail* checkbox and then enter the required e-mail address.

This is because UIs can change a lot, and the more details we describe, the more updating there is later. In general, I find this particular web UI very well designed and easily understandable even without detailed docs.

But this is just a suggestion. If you experience says otherwise, feel free to ignore it :)

Reply:
I actually agree with you. I did, however, copy this from **Generating Host Monitoring Reports** in the _Managing Hosts_ guide. :-)

I'll keep it as is for now and then after this is merged, maybe we change it here and in the _Managing Hosts_ guide in a separate PR.